Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -0,0 +1,993 @@
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.displayVisible = false;
+
+        /**
+         * Gets or sets the flag to show the update section.  This property is observable.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.updateVisible = false;
+
+        /**
+         * Gets or sets the flag to show the logging section.  This property is observable.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.loggingVisible = false;
+
+        /**
+         * Gets or sets the flag to show the style section.  This property is observable.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.styleVisible = false;
+
+        /**
+         * Gets or sets the flag to show the tile info section.  This property is observable.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.tileDebugLabelsVisible = false;
+
+        /**
+         * Gets or sets the JSON for the tileset style.  This property is observable.
+         *
+         * @type {String}
+         * @default '{}'
+         */
+        this.styleString = '{}';
+
+        this._tileset = undefined;
+        this._feature = undefined;
+
+        knockout.track(this, ['performance', 'inspectorVisible', '_statsText', '_pickStatsText', '_editorError', 'showPickStats', 'showStats',
+                              'tilesetVisible', 'displayVisible', 'updateVisible', 'loggingVisible', 'styleVisible', 'tileDebugLabelsVisible', 'styleString', '_feature']);
+
+        this._properties = knockout.observable({});
+        /**
+         * Gets the names of the properties in the tileset.  This property is observable.
+         * @type {String[]}
+         * @readonly
+         */
+        this.properties = [];
+        knockout.defineProperty(this, 'properties', function() {
+            var names = [];
+            var properties = that._properties();
+            for (var prop in properties) {
+                if (properties.hasOwnProperty(prop)) {
+                    names.push(prop);
+                }
+            }
+            return names;
+        });
+
+        var dynamicScreenSpaceError = knockout.observable();
+        knockout.defineProperty(this, 'dynamicScreenSpaceError', {
+            get : function() {
+                return dynamicScreenSpaceError();
+            },
+            set : function(value) {
+                dynamicScreenSpaceError(value);
+                if (defined(that._tileset)) {
+                    that._tileset.dynamicScreenSpaceError = value;
+                }
+            }
+        });
+        /**
+         * Gets or sets the flag to enable dynamic screen space error.  This property is observable.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.dynamicScreenSpaceError = false;
```

Comment:
Throughout this file, please `@see` to the corresponding property in `Cesium3DTileset`.